Calibration/TkAlCaRecoProducers/python/ALCARECOPromptCalibProdSiStripHitEfficiency_cff.py

```diff
@@ -0,0 +1,87 @@
+import FWCore.ParameterSet.Config as cms
+
+# ------------------------------------------------------------------------------
+# configure a filter to run only on the events selected by TkAlMinBias AlcaReco
+import copy
+from HLTrigger.HLTfilters.hltHighLevel_cfi import *
+ALCARECOCalMinBiasFilterForSiStripHitEff = copy.deepcopy(hltHighLevel)
+ALCARECOCalMinBiasFilterForSiStripHitEff.HLTPaths = ['pathALCARECOSiStripCalMinBias']
+ALCARECOCalMinBiasFilterForSiStripHitEff.throw = True ## dont throw on unknown path names
+ALCARECOCalMinBiasFilterForSiStripHitEff.TriggerResultsTag = cms.InputTag("TriggerResults","","RECO")
```

Comment:
```suggestion
ALCARECOCalMinBiasFilterForSiStripHitEff = hltHighLevel.clone(
        HLTPaths = ['pathALCARECOSiStripCalMinBias'],
        throw = True, ## throw on unknown path names
        TriggerResultsTag = ("TriggerResults","","RECO")
)
```
(I also changed the comment to the `thow` parameter...)

Reply:
done at https://github.com/cms-sw/cmssw/pull/37708/commits/5e79159a730dd71c6faf73ff474272440903dace